### PR TITLE
Fixes an undeleted sql query

### PR DESCRIPTION
**Pull request by yoyobatty**
## About The Pull Request
I noticed that one day on a server that is based entirely on TG code this complaint showed up. I looked around for the message the log was complaining about and found this code which looked like it wasn't properly deleting the sql query causing it to complain to the admins.

## Why It's Good For The Game
Because undeleted SQL queries are really really BAD.

## Changelog
:cl: YoYoBatty
code: Fixed an undeleted sql query.
/:cl:

### CODE REVIEW

File: code/modules/client/client_procs.dm [hunk 1]
```diff
@@ -443,7 +443,9 @@ GLOBAL_LIST_EMPTY(external_rsc_urls)
 		return
 	var/sql_ckey = sanitizeSQL(src.ckey)
 	var/datum/DBQuery/query_get_related_ip = SSdbcore.NewQuery("SELECT ckey FROM [format_table_name("player")] WHERE ip = INET_ATON('[address]') AND ckey != '[sql_ckey]'")
-	query_get_related_ip.Execute()
+	if!(query_get_related_ip.Execute())
```
Comment (SpaceManiac) [inline code/modules/client/client_procs.dm]: >`if!(`

are you sure about that

**Review by SpaceManiac — DISMISSED** [review]

File: code/modules/client/client_procs.dm (hunk 1 above)
Comment (yoyobatty) [reply code/modules/client/client_procs.dm]: oof

**Comment by yoyobatty** [comment]
Okay, I'm sure there's more but I don't want to mess with everything...

**Comment by Jordie0608** [comment]
> Because undeleted SQL queries are really really BAD.

Eh... kinda?

**Pull request merged**